satpy/readers/seviri_l1b_native.py

```diff
@@ -567,6 +578,11 @@ def _update_attrs(self, dataset, dataset_info):
         dataset.attrs['standard_name'] = dataset_info['standard_name']
         dataset.attrs['platform_name'] = self.mda['platform_name']
         dataset.attrs['sensor'] = 'seviri'
+        dataset.attrs['nominal_start_time'] = self.header['15_DATA_HEADER'][
+                     'ImageAcquisition']['PlannedAcquisitionTime']['TrueRepeatCycleStart']
+        dataset.attrs['nominal_end_time'] = self.header['15_DATA_HEADER']['ImageAcquisition'][
+            'PlannedAcquisitionTime']['PlannedRepeatCycleEnd']
+
```

Comment:
You could use the "nominal" properties here

Reply:
You mean something like: `dataset.attrs['nominal_start_time'] = self.nominal_start_time`?

Reply:
Updated!